glossary.yml

```
@@ -6136,3 +6182,11 @@
       no lugar de parênteses e vírgulas usados em [JSON](#json). YAML é frequentemente usado em 
       arquivos de configuração e na definição de [parâmetros](#parameter) para vários estilos
       de documentos em [Markdown](#markdown).
+   es:
+    term: "YAML"
```

```suggestion
  es:
   term: "YAML"
```

Fixing indentation errors.

---

Thanks! All checks have passed now, right?

---

Yes, they have. I'll approve them, and @ian-flores can approve the language.